localdev/keycloak/KeycloakClient/kiota-generate.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+echo "Regenerating Keycloak client..."
+dotnet kiota generate --clean-output -o ./src --openapi https://www.keycloak.org/docs-api/latest/rest-api/openapi.json --language csharp -c KeycloakClient -n Keycloak.Client
```

Comment:
Using the latest version can cause unexpected changes.
Better use this link: https://www.keycloak.org/docs-api/26.3.1/rest-api/openapi.json

Reply:
An additional idea, that version can be stored & read from the `.csproj` file.